pkg/Library/NodeGraph/CandidateFinder.js

```diff
@@ -50,9 +51,9 @@ findConnectionCandidates(storeName, {$type}, node, {nodes}, nodeTypes, globalSto
   const candidates = [];
   candidates.push(this.findGlobalCandidate(storeName, globalStores));
   
-  nodes.forEach(n => {
-    if (n.key !== node.key) {
-      candidates.push(this.findCandidateInNode(n.key, $type, nodeTypes[n.type]));
+  keys(nodes).forEach(key => {
```

Comment:
Think this is easier if you iterate the `values`.

Reply:
done